intermediate_source/fx_conv_bn_fuser.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
+"""
+(beta) Building a Conv/BN fuser in FX
+*******************************************************
+**Author**: `Horace He <https://github.com/chillee>`_
+
+In this tutorial, we are going to use FX to do the following:
```

Comment:
link to some FX documentation or provide a brief description?  This just jumps in the deep end.